packages/EasyErrorHandler/src/Bridge/Laravel/Provider/EasyErrorHandlerServiceProvider.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EonX\EasyErrorHandler\Bridge\Laravel\Provider;
+
+use Illuminate\Support\ServiceProvider;
+
+final class EasyErrorHandlerServiceProvider extends ServiceProvider
+{
+    public function boot(): void
+    {
+        $this->publishes([
+            __DIR__ . '/../config/easy-error-handler.php' => \base_path('config/easy-error-hanlder.php'),
```

Comment:
There is a typo in the 2nd part of this line for `hanlder` -> `handler`.

Reply:
thanks. removed `publishes` as this package is not intended to be used in the Laravel.